api/emailLayout.js

```diff
@@ -330,10 +330,10 @@ function ConfirmationEmailLayout() {
   this.paymentInfo = function(paymentInfoData) {
 
     if(!paymentInfoData || Object.keys(paymentInfoData).length === 0) {
-      return "";
+      return ``;
```

Comment:
One suggestion (not a requirement, but idea for future):  Add some logging for this error condition.  Generally if this were to happen something has gone wrong, so seeing seeing even `console.log('Error, unable to send email due to missing data')` would be good.  I don't think robust logging is setup for this project.